aptos-move/vm-genesis/src/lib.rs

```diff
@@ -525,6 +531,31 @@ fn initialize_on_chain_governance(session: &mut SessionExt, genesis_config: &Gen
     );
 }
 
+fn initialize_zkid(session: &mut SessionExt) {
```

Comment:
should we perhaps check the chain id to be non-main?

theoretically in a dooms day scenario if we restarted main, would it load the devnet vk?

Reply:
I think you mean a mainnet wipe, not restart. We will never wipe mainnet or testnet though.